README.md

```diff
@@ -375,19 +397,29 @@ type AreaType = {
   code: string // the municipality or region code (e.g. 482 or 1083)
   enTerm: string // the term used to describe the area in English (e.g. jyllland = jutland)
   region: RegionType // the region that a municipality is located in (e.g. fanø -> syddanmark)
-  altD: string // a possible alternative path for the area in the map
 }
 ```
 
 The types corresponding to each component are:
 
-| Component                    | Name of exported type | Included in type                                            |
+<<<<<<< HEAD
+| Component | Name of exported type | Included in type |
+| ---------------------------- | --------------------- | ------------------------------------------------------ |
+| `Municipalities` | MunicipalityType | { id, name, asciiName, display_name, d, code, region } |
+| `Constituencies`<sup>\*<sup> | ConstituencyType | { id, name, asciiName, display_name, d } |
+| `Regions` | RegionType | { id, name, asciiName, display_name, d, code } |
+| `Islands` | IslandType | { id, name, asciiName, display_name, d, en_term } |
+| `Denmark` | DenmarkType | { id, name, asciiName, display_name, d, en_term } |
+=======
+| Component | Name of exported type | Included in type |
 | ---------------------------- | --------------------- | ----------------------------------------------------------- |
-| `Municipalities`             | MunicipalityType      | { id, name, asciiName, displayName, d, code, region, altD } |
-| `Constituencies`<sup>\*<sup> | ConstituencyType      | { id, name, asciiName, displayName, d }                     |
-| `Regions`                    | RegionType            | { id, name, asciiName, displayName, d, code }               |
-| `Islands`                    | IslandType            | { id, name, asciiName, displayName, d, enTerm }             |
-| `Denmark`                    | DenmarkType           | { id, name, asciiName, displayName, d, enTerm }             |
+| `Municipalities` | MunicipalityType | { id, name, asciiName, displayName, d, code, region, altD } |
+| `Constituencies`<sup>\*<sup> | ConstituencyType | { id, name, asciiName, displayName, d } |
+| `Regions` | RegionType | { id, name, asciiName, displayName, d, code } |
+| `Islands` | IslandType | { id, name, asciiName, displayName, d, enTerm } |
+| `Denmark` | DenmarkType | { id, name, asciiName, displayName, d, enTerm } |
+
+> > > > > > > 808b49f367902cff4efc069a900a1a3fbc691aeb
```

Comment:
Same case as above here. Again, I recommend you just replace it with the old text here. Also, nice catch seeing that `altD` should be removed here 🥇